Add Troubleshooting section in README (#186)
awslabs/smithy-typescript: README.md

You can find the build artifacts of the test package at:
`build/smithyprojections/smithy-typescript-codegen-test/source/typescript-codegen`

## Troubleshooting

> Most of the issues with Gradle can be fixed by stopping the daemon by running `./gradlew --stop`

Reviewer: ```suggestion
Many Gradle issues can be fixed by stopping the daemon by running `./gradlew --stop`
```
